sklego/meta.py

```diff
@@ -54,6 +54,52 @@ def __init__(self, estimator, groups, use_fallback=True):
         self.groups = groups
         self.use_fallback = use_fallback
 
+    def __check_group_cols_exist(self, X):
+        try:
+            x_cols = set(X.columns)
+        except AttributeError:
+            try:
+                ncols = X.shape[1]
+            except IndexError:
```

Comment:
is `X` ever only 1d?

Reply:
I guess it could be. groupby-mean as a model?